### PR TITLE
follow up removal of all things enum in contrib/ dir

### PR DESCRIPTION
**Pull request by shintasmith**
Remove enums from grinder and other things under contrib

### CODE REVIEW

**Comment by shintasmith** [comment]
@usnavi @ChandraAddala @izrik pls review. This PR removes Enum from the old grinder.

File: contrib/grinder/properties/grinder.properties.perf01 [hunk 1]
```diff
@@ -2,30 +2,24 @@ grinder.script=../scripts/grinder.py
 grinder.package_path=/usr/local/lib/python2.7/dist-packages
 grinder.runs=0
 grinder.consoleHost=10.223.208.207
-grinder.threads=145
+grinder.threads=125
```
Comment (ChandraAddala) [inline contrib/grinder/properties/grinder.properties.perf01]: Why did you change the threads?
